python/rikai/types/vision.py

```diff
@@ -59,6 +59,8 @@ def from_array(
         array: np.ndarray,
         uri: Union[str, Path],
         mode: str = None,
+        format: str = None,
```

Comment:
yea, here it uses `format`.  Lets be consist with `format` and `**kwargs`?

Reply:
this is API for the image itself and keeping consistency with PIL.Image makes sense, but I think we don't necessarily need that for the video_to_image function